### PR TITLE
replace simple-argv with yargs

### PR DESCRIPTION
**Pull request by cfloress**

### CODE REVIEW

**Review by jossemargt-cto-ai — APPROVED** [review]

File: src/vault.ts
Comment (klebertertulino) [inline src/vault.ts:1]: Why these changes @cfloress?

**Review by klebertertulino — APPROVED** [review]
LGTM

**Review by vorticist — APPROVED** [review]

File: src/vault.ts [hunk 1]
```diff
@@ -6,7 +6,9 @@ const pexec = util.promisify(oexec);
 
 const slice = process.argv[2] === '.' ? 3 : 2
 const ARGS = process.argv.slice(slice);
-const OPTIONS = require('simple-argv')
+const yargs = require('yargs/yargs')
+const { hideBin } = require('yargs/helpers')
```
Comment (marcoguillenuio) [inline src/vault.ts:10]: @cfloress Could it be standardized to have the constants names in the same way (common convention is uppercase)

Comment (cfloress) [reply src/vault.ts:10]: Hi @marcoguillenuio , Jossemar's answer applies --> https://github.com/workflows-sh/aws-eks-ec2-asg-cdk/pull/16#discussion_r1213265368 , thanks @jossemargt-cto-ai

**Comment by kc-dot-io** [comment]
👍 LGTM - please publish to workflows team and then commit the ops.yml with updated version to this PR for merge!
